spec/support/saml_auth_helper.rb

```diff
@@ -31,11 +31,10 @@ def saml_settings
   end
 
   def request_authn_context
```

Comment:
since this now returns an array, WDYT of renaming it to be plural?

```suggestion
  def request_authn_contexts
```

Reply:
Sounds good - updated in 3efab5f